po/sr_ME@latin.po

```diff
@@ -0,0 +1,566 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
```

Comment:
The initial section has to be filled in, with the standard GC header (look in the po folder) and your contact info (for attribution).